include/sgct/config.h

```diff
@@ -361,6 +361,13 @@ struct Cluster {
 };
 void validateCluster(const Cluster& cluster);
 
+struct GeneratorVersion {
+    std::string generatorName;
```

Comment:
should we just call the member "name" instead?

Reply:
I'll change it to "name"